com.github.tsmetana.mpk3-settings.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.github.tsmetana.mpk3-settings
```

Comment:
id MUST start with `io.github` for GitHub based IDs.

Reply:
There are a lot of apps with com.github... ID

Reply:
two wrongs don't make a right. The docs always said otherwise and we started enforcing this last year.

This also need to be changed upstream, including the GtkApplication initialisation.

Reply:
Created a PR to change it, but it takes time.

Reply:
fixed